PySoar/importClass.py

```diff
@@ -13,24 +14,30 @@ class SoaringSpotImport(object):
     def __init__(self, url, download_progress):
```

Comment:
In fact I also thought about writing a general input interface but I tried to stay as close to the master branch to facilitate the merge